### PR TITLE
v1 of returning amount of time left in wayup swing

### PR DESCRIPTION
**Pull request by emilymaskin**
feel free to refactor

### CODE REVIEW

File: gendo.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 import os
```
Comment (johnbenjaminlewis) [inline gendo.py:3]: I support separating stdlib imports from dependency imports and imports from this repo. Like this:

``` python
from datetime import date, datetime, timedelta
import os

from pyfieri.guy_fieri import speak
import requests

from gendo import Gendo
```

@nficano thoughts?

Comment (johnbenjaminlewis) [inline gendo.py:3]: Also you should `import calendar` from the standard library to use day of week constants. See below

File: gendo.py [hunk 2]
```diff
@@ -39,3 +40,37 @@ def quote_of_the_day():
...
+    now = datetime.now()
+
+    start_of_day = now.replace(hour=9, minute=30)
+    end_date = date(2016, 1, 30)
+    time_remaining = timedelta(days=0)
+
+    d = today
+    while d <= end_date:
+        day_of_week = d.weekday()
+        if day_of_week < 5:
```
Comment (johnbenjaminlewis) [inline gendo.py:57]: use `calendar` here:

``` python
if day_of_week < calendar.SATURDAY:
  ...
```

Comment (emilymaskin) [inline gendo.py:57]: just curious, is that better for any reason other than readability?

Comment (johnbenjaminlewis) [inline gendo.py:57]: Ps, indexing time from 0 is confusing. I first read this as if day_of_week < friday

Comment (johnbenjaminlewis) [inline gendo.py:57]: Readability

**Pull request closed without merging**